victron_ble/devices/smart_lithium.py

```diff
@@ -0,0 +1,91 @@
+from typing import Optional
+
+from construct import Array, BitsInteger, BitStruct, ByteSwapped, Padding
+
+from victron_ble.devices.base import Device, DeviceData
+
+
+class SmartLithiumData(DeviceData):
+    def get_bms_flags(self) -> int:
+        """
+        Get the raw bms_flags field (meaning not documented).
+        """
+        return self._data["bms_flags"]
+
+    def get_error_flags(self) -> int:
+        """
+        Get the raw error_flags field (meaning not documented).
+        """
+        return self._data["error_flags"]
+
+    def get_battery_voltage(self) -> Optional[float]:
+        """
+        Return the voltage in volts
+        """
+        return self._data["battery_voltage"]
+
+    def get_battery_temperature(self) -> int:
+        """
+        Return the temperature in Celsius if the aux input is set to temperature
+        """
+        return self._data["battery_temperature"]
+
+    def get_cell_voltages(self) -> list:
+        """
+        Return the voltage of each cell (floats where -inf is <2.61V, +inf is >3.85V, None is N/A)
+        """
+        return self._data["cell_voltages"]
+
+    def get_balancer_status(self) -> int:
+        """
+        Get the raw balancer_status field (meaning not documented).
+        """
+        return self._data["balancer_status"]
+
+
+class SmartLithium(Device):
+    data_type = SmartLithiumData
+
+    # https://community.victronenergy.com/questions/187303/victron-bluetooth-advertising-protocol.html
+    # Reverse the entire packet, because non-aligned integers are packed
+    # little-endian
+    PACKET = ByteSwapped(
+        BitStruct(
+            Padding(1),  # unused
+            "battery_temperature" / BitsInteger(7),  # -40..86C
+            "balancer_status" / BitsInteger(4),  # 0 = Imbalance, 1 = balanced, 3 = Imbalance
```

Comment:
From testing, I'm pretty sure that 0="Unknown"

Reply:
```suggestion
            "balancer_status" / BitsInteger(4),  # 0 = unknown, 1 = balanced, 3 = imbalance
```